backend/src/v2/driver/driver.go

```diff
@@ -164,7 +162,7 @@ func RootDAG(ctx context.Context, opts Options, mlmd *metadata.Client) (executio
 		pipelineRoot = cfg.DefaultPipelineRoot()
 		glog.Infof("PipelineRoot=%q from default config", pipelineRoot)
 	}
-	pipelineBucketSessionInfo, err = cfg.GetBucketSessionInfo(pipelineRoot)
+	pipelineBucketSessionInfo, err = cfg.GetStoreSessionInfo(pipelineRoot)
```

Comment:
Can you take a look at this [until line 175](https://github.com/rimolive/data-science-pipelines/blob/rebase-2.2.0/backend/src/v2/driver/driver.go#L158-L175) and change bucketsession to match upstream's storeSessionInfo: https://github.com/kubeflow/pipelines/blob/2.2.0/backend/src/v2/driver/driver.go#L148-L165